server-common/src/main/java/org/apache/kafka/server/config/ServerLogConfigs.java

```diff
@@ -136,14 +136,16 @@ public class ServerLogConfigs {
     public static final String MIN_IN_SYNC_REPLICAS_CONFIG = ServerTopicConfigSynonyms.serverSynonym(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG);
     public static final int MIN_IN_SYNC_REPLICAS_DEFAULT = 1;
     public static final String MIN_IN_SYNC_REPLICAS_DOC = "When a producer sets acks to \"all\" (or \"-1\"), " +
```

Comment:
Hmm, is there a reason why this cannot reference the definition in `TopicConfig`?